vllm/v1/executor/multiproc_executor.py

```diff
@@ -586,6 +586,8 @@ def monitor_parent_death():
 
             if ready_writer is not None:
                 logger.exception("WorkerProc failed to start.")
+            elif shutdown_event.is_set():
```

Comment:
@njhill Since `except Exception` can catch `RuntimeError`,  only need to check `shutdown_event.is_set()` to avoid logging this error.